qualtran/bloqs/multi_control_multi_target_pauli.py

```diff
@@ -137,6 +140,160 @@ def _has_unitary_(self) -> bool:
         return True
 
 
+@frozen
+class MultiControlX(Bloq):
+    r"""Implements multi-control, single-target X gate as a bloq using $n-1$ clean ancillas.
+
+    References:
+        [Constructing Large Controlled Nots]
+        (https://algassert.com/circuits/2015/06/05/Constructing-Large-Controlled-Nots.html)
```

Comment:
what's the relevant section of this? The final construction looks like it uses 1 ancilla but above you say n-1 ancilla